timescaledb/overview/release-notes/index.md

```diff
@@ -32,22 +32,48 @@ If you have questions about distributed hypertables, join our #multinode channel
 on [community Slack](https://slack.timescale.com/) for installation details and
 follow these [setup instructions][distributed-hypertables-setup].
 
-### What's new in TimescaleDB 2.6.1:
+### What's new in TimescaleDB 2.7:
 
-- Continuous aggregates with compression
-- `time_bucket_ng` support for N months, years and timezones on continuous aggregates
 
-You can read more about this release on our [blog post](https://tsdb.co/timescaledb-2-6).
-This release also contains bug fixes since the 2.6.0 release.
+You can recreate a cagg to benefit from the improvement with the caveat that the original data need to exist  
```

Comment:
```suggestion
You can recreate a continuous aggregate to benefit from the improvement, if the
original data already exists.
```

Reply:
Maybe we should say "still exists"? Because I think the biggest concern is trying to recreate a cagg over data dropped in a retention policy (whereas if the data _never_ existed, well you have no cagg anyway 😛 )